workitem/workitem_repository.go

```diff
@@ -26,15 +26,60 @@ import (
 
 const orderValue = 1000
 
+// DirectionType represents execution order direction
 type DirectionType string
 
+// Possible values for execution order direction
 const (
 	DirectionAbove  DirectionType = "above"
 	DirectionBelow  DirectionType = "below"
 	DirectionTop    DirectionType = "top"
 	DirectionBottom DirectionType = "bottom"
 )
 
+// SortWorkItemsBy is type used to define parameters using which workitems will be sorted
+type SortWorkItemsBy string
+
+// Available sort orders
+var (
+	SortWorkItemsByExecutionAsc  = SortWorkItemsBy("execution_order ASC")
+	SortWorkItemsByExecutionDesc = SortWorkItemsBy("execution_order DESC")
+	SortWorkItemsByCreatedAtAsc  = SortWorkItemsBy("created_at ASC")
+	SortWorkItemsByCreatedAtDesc = SortWorkItemsBy("created_at DESC")
+	SortWorkItemsByUpdatedAtAsc  = SortWorkItemsBy("updated_at ASC")
+	SortWorkItemsByUpdatedAtDesc = SortWorkItemsBy("updated_at DESC")
+	SortWorkItemsByDefault       = SortWorkItemsByExecutionDesc
+)
+
+// ParseSortWorkItemsBy parses the string input and returns object of type SortWorkItemsBy
+// which can directly be used while querying database to order the output.
+func ParseSortWorkItemsBy(s *string) (SortWorkItemsBy, error) {
+	if s == nil {
+		// this is the default case
+		// which returns recently updated spaces first
```

Comment:
`recently updated spaces`? 

The default case would return workitems which have highest execution order

Reply:
Fixed in fc4532d72dbae3555f995aec00b4266cfc524cf2